rdr_service/genomic/genomic_job_components.py

```diff
@@ -2543,6 +2638,32 @@ def _get_new_c1_participants(self, from_date):
 
         return list([list(r) for r in zip(*result)])
 
+    def _get_long_read_participants(self):
+        """
+        Retrieves participants based on filters that have
+        been denoted to use in the long read pilot program
+        """
+        with self.member_dao.session() as session:
+            result = session.query(GenomicSetMember).join(
+                ParticipantSummary,
+                GenomicSetMember.participantId == ParticipantSummary.participantId,
+            ).join(
+                ParticipantRaceAnswers,
+                ParticipantRaceAnswers.participantId == ParticipantSummary.participantId,
+            ).join(
+                Code,
+                ParticipantRaceAnswers.codeId == Code.codeId,
+            ).filter(
+                Code.value == 'WhatRaceEthnicity_Black',
+                GenomicSetMember.genomeType == 'aou_wgs',
+                GenomicSetMember.genomeType != 'long_read',
+                GenomicSetMember.genomicWorkflowState != GenomicWorkflowState.IGNORE,
+                ParticipantSummary.participantOrigin == 'vibrent',
+                ParticipantSummary.ehrUpdateTime.isnot(None)
+            ).all()
```

Comment:
Will the number of participants being used for a long_read manifest always be limited to a cap (set to 384 where these participants are processed)? Would it work to have a limit clause on this query (rather than returning all GenomicSetMembers that match the criteria)? That could help with performance if we aren't expecting to use them all.

Reply:
Yea sounds good. Will add it. Still getting familiar with the ORM.